zarr/meta.py

```diff
@@ -201,6 +206,10 @@ def encode_fill_value(v, dtype):
         return int(v)
     elif dtype.kind == 'b':
         return bool(v)
+    elif dtype.kind in 'c':
+        v = (encode_fill_value(v.real, dtype.type().real.dtype),
+             encode_fill_value(v.imag, dtype.type().imag.dtype))
+        return v
```

Comment:
I take it this encoding / decoding of fill values is necessary is necessary because json does not support serialization of complex data types?

Reply:
That's right.

This will encode to a JSON list, which is nice for readability. Though we could certainly imagine other ways of handling this (e.g. base64 encoded string).

Any preferences?

Reply:
That seems totally reasonable.